tests/ci/run_build.py

```diff
@@ -237,9 +237,19 @@ def build_srpm_and_rpm(
 failed = []
 rebuild_success = []
 total = 0
+docs_spec_executed = False
 
 for spec in args.specfiles:
-    if not spec.endswith('.spec'):
+    if 'components/admin/docs/SPECS/docs.spec' == spec:
```

Comment:
Ah comment here would be nice. My understanding is that this tries to do a test build if any file changed in the `docs/` tree, right?

Reply:
The idea is to not build the docs more than once if `docs.spec`, `components/admin/docs/SOURCES/**` and any file in `docs/recipes/install/**` are modified